internal/service/cloudbackupschedule/resource_cloud_backup_schedule.go

```diff
@@ -492,9 +492,9 @@ func cloudBackupScheduleCreateOrUpdate(ctx context.Context, connV2 *admin.APICli
 	}
 
 	req := &admin.DiskBackupSnapshotSchedule{}
-
-	if v, ok := d.GetOk("copy_settings"); ok && len(v.([]any)) > 0 {
-		req.CopySettings = expandCopySettings(v.([]any))
+	copySettings := d.Get("copy_settings")
+	if len(copySettings.([]any)) > 0 || d.HasChange("copy_settings") {
+		req.CopySettings = expandCopySettings(copySettings.([]any))
```

Comment:
copySettings.([]any) could panic if conversion can't be done, it's normally better to check for errors